components/google-cloud/google_cloud_components/mb_sdk_component/__init__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 Google LLC. All Rights Reserved.
```

Comment:
Flagging this to note we may change the name of this directory before release.

Reply:
changed to `aiplatfrom_component`